python/tvm/relay/frontend/tflite.py

```diff
@@ -1089,7 +1093,7 @@ def convert_square(self, op):
 
         return out
 
-    def _convert_elemwise(self, relay_op, op):
+    def _convert_elemwise(self, relay_op, op, use_real_qnn=True):
```

Comment:
Why is `use_real_qnn` required? I would suggest to move the same_qnn_params test L1225 and L1229 and keep _convert_elemwise unchanged.

Reply:
Ping on this.

Reply:
"use_real_qnn=False" allows __convert_elemwise_ to use non-quantized version of the operation if all supplied parameters and output of the operation have same quantization values. Some other quantized tflite operations also supposed to use this feature

Reply:
Can we skip the use_real_qnn by moving the check to L1225 and L1229 and keeping _convert_elemwise unchanged? Adding using_real_qnn seems little adhoc.

Reply:
The extraction of "use_real_qnn" functionality to _convert_minimum and _convert_maximum methods (L1225 and L1229) will lead either: to change to operation's parameters stripping the qnn_attrs from at least lhs input tensor or to addition an extra flag forces _convert_elemwise to use non quantized version of the operation. Alternatively I might not understand your point here.

Reply:
I might be not correct here, but the whole idea of using same qnn parameters is about being able to re-use non-quantized version of the operation. In case of Slice op #6217 there is only a check and the non-quantized operation is always used. In case of maximum and minimum this is not possible without changes either in the command operands (the qnn_params should stripped off) or, alternatively changes in _convert_elemwise in order explicitly prevent it going via quantized version.

Reply:
That makes sense now. Thanks for your patience. I would suggest to rename the `use_real_qnn` to `ignore_qnn_params`.

Reply:
renamed